test/timezone/lookup/test_google.rb

```diff
@@ -5,18 +5,18 @@
 require 'timecop'
 require_relative '../../http_test_client'
 
-class TestGoogle < ::Minitest::Test
+class TestGoogle < Minitest::Test
   parallelize_me!
 
   def coordinates
     [-34.92771808058, 138.477041423321]
   end
 
-  def lookup(body = nil, &_block)
+  def lookup(body = nil, &block)
     config = OpenStruct.new
     config.api_key = 'MTIzYWJj'
     config.request_handler = HTTPTestClientFactory.new(body)
-    yield config if block_given?
+    yield config if block
```

Comment:
Ditto here.

Reply:
This is a Rubocop fix - https://msp-greg.github.io/rubocop-performance/RuboCop/Cop/Performance/BlockGivenWithExplicitBlock.html

I'm not sure why the previous version would be preferable.